build.sbt

```diff
@@ -7,14 +7,15 @@ lazy val root = project.in(file("."))
   .dependsOn(octopusJVM, octopusJS, octopusCatsJVM, octopusCatsJS, octopusScalazJVM, octopusScalazJS)
 
 lazy val versions = new {
-  val scala212 = "2.12.8"
+  val scala213 = "2.13.1"
+  val scala212 = "2.12.10"
```

Comment:
Let's update 2.12 to the latest recently published - 2.12.11

Reply:
I think that may cause us to bump Scala.js to 1.0, but as long as everything works, I'm fine with that.
Otherwise we may just last release for sjs0.6, then bump to 1.0 in a separate PR.